pkgs/applications/graphics/gimp/wrapper.nix

```
@@ -43,5 +43,9 @@ symlinkJoin {
     done
   '';
 
-  inherit (gimp) meta;
+  meta = gimp.meta // {
+    description = "${gimp.meta.description} with plugins: ${
+      lib.concatMapStringsSep ", " (p: p.pname) selectedPlugins
+    }";
+  };
```

May be too long for the one-line description? How about (haven't tried to build/eval with it):
```suggestion
  meta = gimp.meta // {
    description = "${gimp.meta.description} with plugins";
    longDescription = "Includes the following plugins: ${
      lib.concatMapStringsSep ", " (p: p.pname) selectedPlugins
    }.";
  };
```